engine/src/io/data_parser/GDFParser.cpp

```diff
@@ -20,33 +20,34 @@ gdf_parser::gdf_parser(std::vector<cudf::column *> columns_, std::vector<std::st
 
 gdf_parser::~gdf_parser() {}
 
-std::unique_ptr<ral::frame::BlazingTable> gdf_parser::parse(
+ral::frame::TableViewPair gdf_parser::parse(
```

Comment:
No this cant be left off. This must be implemented here.

Reply:
This will be implemented once @Christian8491 's PR is complete which has the updated Schema object that contains a BlazingTableView

Reply:
https://github.com/BlazingDB/blazingsql/pull/268